### PR TITLE
Add minutes of 20221114 container meeting

### PR DESCRIPTION
**Pull request by joshmue**
Signed-off-by: Joshua Mühlfort <muehlfort@gonicus.de>

### CODE REVIEW

File: container/20221114.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  4. Generally: Sharing data between pods
+  5. Generally: Sharing application code between pods (often used when LAMP apps are transitioned to k8s)
+* Implementation options
+  - NFS
+  - cephfs
+  - manila: ganesha NFS, via userspace service (not fuse) (https://github.com/nfs-ganesha/nfs-ganesha/wiki)
+* PoC for SCS operator from 23tech: 
+    - Minutes PR (https://github.com/SovereignCloudStack/minutes/pull/74) is wrong place for discussion, so will open new issue (AI @garloff)
+    - Would be nice to have PR reviewed
+* Conformance test framework discussion. Two different main scenarios:
```
Comment (garloff) [inline container/20221114.md:37]: I think I ended up with three different test scenarios:
1. Conformance tests for SCS REST API (Cluster Lifecycle Management)
2. Conformance tests for Cluster Properties (Assume cluster already exists and we have cluster-admin access)
3. Implementation-specific tests (unit tests / integration tests), which may assume CAPI (in our ref. impl.)

File: container/20221114.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    1. Test creation/changing/... clusters via SCS standardized interfaces (the REST interface we are discussing)
+       - (Cluster API specific): Cluster API test framework.
+         * https://github.com/syself/cluster-api-provider-hetzner/tree/main/test/e2e
+         * https://github.com/syself/cluster-api-provider-hetzner/blob/main/test/e2e/conformance_test.go
+         * https://github.com/kubernetes-sigs/cluster-api/blob/fa70f61a77828cea2d5d059ffd0dfcf6bcbc9f2d/test/e2e/k8s_conformance.go#L109
```
Comment (garloff) [inline container/20221114.md:42]: As far as *SCS conformance* tests are concerned, we can not assume that the implementation uses CAPI.
(That said, the tests that are part of the CAPI tests may not make that assumption everywhere ... at least some comment from Janis indicated so.)

**Review by garloff — CHANGES_REQUESTED** [review]

File: container/20221114.md (hunk 1 above)
Comment (joshmue) [reply container/20221114.md:37]: I took your new bullet points into the minutes directly :+1:

File: container/20221114.md (hunk 2 above)
Comment (joshmue) [reply container/20221114.md]: Moved that part to the new 3rd scenario.

**Review by garloff — APPROVED** [review]
Thanks, let's merge this!

**Pull request merged**